coordinated-entry-screening/static/css/custom.css

```diff
@@ -1,6 +1,66 @@
+:root {
+    --blue: #0178BC;
+    --light-blue: #4da0d0;
+}
+
+/*http://getbootstrap.com/docs/4.0/examples/sticky-footer/*/
+html {
+  position: relative; 
+  min-height: 100%;
+}
+
+body {
+    font-family: 'Libre Franklin', sans-serif !important;
+    margin-bottom: 60px;
+}
+
+h1, h2, h3, h4, h5, h6 {
+    font-weight: 800;
+}
```

Comment:
If you wanted, this could be a different font. The change in weight doesn't really come across. It could also be `text-transform: uppercase`...neither are necessary, just if you wanted to play around.

Reply:
It would take me the rest of the morning to chose a secondary font! I'd love to learn any tips you have for expediting the font- and color-selection process (i.e., when the client does not have something in mind).

`uppercase` works well.

Reply:
For finding an initial typeface, I inspect the page I'm hoping to mimic (and see that CSH uses Google's Roboto) if I want to directly copy. Google Fonts has a quick tool where you can see popular pairings, at the bottom of each specimen: https://fonts.google.com/specimen/Libre+Franklin

In general, I always skip two font weights. For example, if you were using "Light" for paragraph text, I would use "Medium" for bolding paragraph text (by manually setting the `font-family` on `p strong` in CSS), and "Bold" or "Extra-Bold" for headers, depending on how it looks together.

I also mix sans-serif and serif; if you have a sans-serif typeface for paragraphs (like here), I'd find a serif for headers. Google Fonts has Roboto Slab as a popular pairing, so I'd use that if I wanted to go with two different typefaces.